build.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineBuildConfig } from 'unbuild'
+
+export default defineBuildConfig({
+  entries: [
+    {
+      builder: 'mkdist',
+      input: './src/templates/',
+      outDir: './dist/templates',
+      ext: 'js',
+      declaration: false,
+    },
+  ],
+  externals: [
+    '@sentry/webpack-plugin',
+    'webpack',
+    '@sentry/tracing',
+    '@sentry/types',
+    '@sentry/browser',
+    '@sentry/node',
+    '@sentry/cli',
+    '@sentry/minimal',
+    'vuex',
+    '@nuxt/types',
+  ],
```

Comment:
Since `unbuild` documentation is lacking, can you educate me on why this is needed?

Reply:
So I'll be honest and just say that I'm in the same boat. My understanding is that it should be something akin to the types on a tsconfig and are used during the build process for typechecking but I'm not sure.
I just kept adding untill there were no more errors in the build

Reply:
I found this message from @harlan-zw on the Nuxt discord as some other users was asking about the externals option

> well when you have it as a dev dependency it means the end user won't install it, so your package wouldn't work
> to try and resolve that, nuxt-module-builder (unbuild) presumes that you want to instead inline the dependency (combining the dependencies code with your module code)
> it's really useful when you are depending on a third party package that you only need some of the code from as it tree-shakes anything not needed

Since the only imports from those packages are types I assume unbild is not inlining anything anyway

Reply:
Is it possible to globally disable that instead of having to explicitly exclude? I can't imagine any case when I would want to include dev dependency in built output.